src/script/components/MessagesList/Message/ContentMessage/asset/TextMessageRenderer.tsx

```diff
@@ -66,98 +66,49 @@ export const TextMessageRenderer: FC<TextMessageRendererProps> = ({
     if (!containerRef.current) {
       return undefined;
     }
-    const emailLinks = containerRef.current && [...containerRef.current.querySelectorAll('[data-email-link]')];
-    const linkTargets = containerRef.current && [...containerRef.current.querySelectorAll('a[data-md-link]')];
-    const msgLinkTargets = containerRef.current && [...containerRef.current.querySelectorAll('[data-uie-name]')];
+    const emailLinks =  [...containerRef.current.querySelectorAll('[data-email-link]')];
+    const markdownLinkTargets = [...containerRef.current.querySelectorAll('[data-md-link]')];
     const hasMentions = asset && asset.mentions().length;
     const msgMention = hasMentions
-      ? containerRef.current && [...containerRef.current.querySelectorAll('.message-mention')]
-      : null;
-
-    // set tabindex for each interactive element based on the element focus state
-    if (msgMention) {
-      msgMention.forEach(mention => {
-        mention.setAttribute('tabindex', isCurrentConversationFocused ? '0' : '-1');
-      });
-    }
-
-    if (linkTargets.length) {
-      linkTargets.forEach(link => {
-        link.setAttribute('tabindex', isCurrentConversationFocused ? '0' : '-1');
-      });
-    }
-
-    if (msgLinkTargets.length) {
-      msgLinkTargets.forEach(link => {
-        link.setAttribute('tabindex', isCurrentConversationFocused ? '0' : '-1');
-      });
-    }
-
-    if (emailLinks.length) {
-      emailLinks?.forEach(emailLink => {
-        emailLink.setAttribute('tabindex', isCurrentConversationFocused ? '0' : '-1');
-      });
-    }
+      ? [...containerRef.current.querySelectorAll('.message-mention')]
+      : [];
 
     const handleKeyEvent = (event: KeyboardEvent, elementType: ElementType) => {
       if (isCurrentConversationFocused) {
         handleKeyDown(event, () => onMessageClick(asset, event, elementType));
       }
     };
 
-    const handleClickEmail = (event: Event) => {
-      return event.type === 'keydown'
-        ? handleKeyEvent(event as KeyboardEvent, 'email')
-        : onMessageClick(asset, event as MouseEvent, 'email');
-    };
-    emailLinks?.forEach(emailLink => {
-      events.forEach(eventName => {
-        emailLink.addEventListener(eventName, handleClickEmail);
-      });
-    });
-
-    const handleClickLink = (event: Event) => {
-      return event.type === 'keydown'
-        ? handleKeyEvent(event as KeyboardEvent, 'markdownLink')
-        : onMessageClick(asset, event as MouseEvent, 'markdownLink');
-    };
-
-    linkTargets?.forEach(msgLink => {
-      events.forEach(eventName => {
-        msgLink.addEventListener(eventName, handleClickLink);
-      });
-    });
-
-    const handleClickMention = (event: Event) => {
+    const handleMsgEvent = (event: Event) => {
       return event.type === 'keydown'
         ? handleKeyEvent(event as KeyboardEvent, 'mention')
         : onMessageClick(asset, event as MouseEvent, 'mention');
```

Comment:
It seems you are always sending a `mention` click even when clicking on a link or an email. Am I wrong?